CONTRIBUTING.md

```diff
@@ -154,3 +154,30 @@ These are the steps to create a new release:
    eventually too).
 
 7. Celebrate!
+
+##  Cross-Arch Testing
+
+This project has built-in support for testing across multiple architectures.
+Currently, our CI conducts tests on `arm64` machines using QEMU emulation. We
+also have the flexibility to expand this support to include additional
+architectures in the future.
+
+This project containers Dockerfiles that can be used in the CI to test the
```

Comment:
hmm I thought we fixed in repo-config, it should be `contains` instead of `containers`

Reply:
Yeah, I just didn't regenerate the templates yet :(

I'll do it later today.